presto-docs/src/main/sphinx/security/built-in-system-access-control.rst

```diff
@@ -77,13 +77,15 @@ contents:
 
 The config file is specified in JSON format.
 
-* It contains the rules defining which catalog can be accessed by which user (see Catalog Rules below).
+* The catalog rules specifying which catalog can be accessed by which user (see Catalog Rules below).
+* The schema rules specifying which users have ownership of which schemas (see Schema Rules below).
+* The table rules specifying which privileges are granted to which users over which tables (see Table Rules below).
```

Comment:
This is very "wordy". Any simpler suggestions?

Reply:
Yes .. something like that .. make sure you have all anchors in place and such.

```
It contains rules that define which use has access to specific resources:

* :ref:`Catalogs <catalog-rules>` 
* :ref:`Schemas <schema-rules>`
```

Reply:
Done.